packages/Pager/src/pager.scss

```diff
@@ -0,0 +1,189 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-pager-font-family: #{tokens.$primary-font-family};
+  --ids-pager-font-size: #{tokens.$font-size-4};
+  --ids-pager-height: #{tokens.$space-5};
+  --ids-pager-line-height: #{tokens.$line-height-xxxl};
+
+  /* Item */
+  --ids-pager-item-padding: #{tokens.$space-1};
+  --ids-pager-item-background: #{tokens.$samoyed};
+  --ids-pager-item-border: 1px solid #{tokens.$grey-300};
+  --ids-pager-item-prev-border-radius: #{tokens.$border-radius-sm} 0 0 #{tokens.$border-radius-sm};
+  --ids-pager-item-next-border-radius: 0 #{tokens.$border-radius-sm} #{tokens.$border-radius-sm} 0;
+  --ids-pager-item-color: #{tokens.$grey-800};
+  --ids-pager-item-z-index: #{tokens.$z-index-10};
+
+  /* Selected Item */
+  --ids-pager-item-selected-background: #{tokens.$electric-blue-500};
+  --ids-pager-item-selected-color: #{tokens.$samoyed};
+
+  /* Hover */
+  --ids-pager-item-hover-border: 1px solid #{tokens.$grey-600};
+
+  /* Focus */
+  --ids-pager-item-focus-border: 1px solid #{tokens.$electric-blue-600};
+
+  /* Disabled */
+  --ids-pager-item-disabled-background: #{tokens.$grey-300};
+  --ids-pager-item-disabled-border: 1px solid #{tokens.$grey-300};
+  --ids-pager-item-disabled-color: #{tokens.$grey-600};
+
+  /* Arrows */
+  --ids-pager-arrow-color: #{tokens.$grey-500};
+
+  /* Ellipsis */
+  --ids-pager-ellipsis-color: #{tokens.$grey-600};
+
+  /* Results */
+  --ids-pager-results-color: #{tokens.$grey-600};
+  --ids-pager-results-margin: 0 0 0 #{tokens.$space-3};
+}
+
+%focus-style {
+  border: var(--ids-pager-item-focus-border);
+  z-index: var(--ids-pager-item-z-index);
+}
+
+%hover-style {
+  border: var(--ids-pager-item-hover-border);
+  z-index: var(--ids-pager-item-z-index);
+}
+
+.ids-pager {
+  align-items: center;
+  display: flex;
+  font-family: var(--ids-pager-font-family);
+  font-size: var(--ids-pager-font-size);
+  line-height: var(--ids-pager-line-height);
+
+  &,
+  & * {
+    box-sizing: border-box;
+  }
+}
+
+.ids-pager__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ids-pager__item {
+  display: flex;
+  flex: 0 0 auto;
+}
+
+.ids-pager__button {
+  align-items: center;
+  appearance: none;
+  background: var(--ids-pager-item-background);
+  border: var(--ids-pager-item-border);
+  color: var(--ids-pager-item-color);
+  cursor: pointer;
+  display: flex;
+  flex: 0 0 auto;
+  font-family: var(--ids-pager-font-family);
+  font-size: var(--ids-pager-font-size);
+  height: var(--ids-pager-height);
+  justify-content: center;
+  line-height: var(--ids-pager-line-height);
+  margin: 0 0 0 -1px;
+  min-width: 3.2rem;
+  outline: 0;
+  padding: var(--ids-pager-item-padding);
+  position: relative;
+
+  &:hover {
+    @extend %hover-style;
+  }
+
+  &:disabled {
```

Comment:
There is still a small glitch in the hover of the preview button. To fix this you just have to set the `z-index: 0;`

 

https://user-images.githubusercontent.com/2379339/196772535-301ab5f1-2ba2-447f-9181-e6407e04293b.mov